src/components/NumberInput/NumberInput.jsx

```diff
@@ -69,20 +81,24 @@ class NumberInput extends React.Component {
 }
 
 NumberInput.propTypes = {
-  placeholder: PropTypes.string,
-  minValue: PropTypes.number,
+  ...getDataProps(),
+  disabled: PropTypes.bool,
+  error: PropTypes.bool,
   maxValue: PropTypes.number,
+  minValue: PropTypes.number,
+  onBlur: PropTypes.func,
+  onChange: PropTypes.func.isRequired,
+  onFocus: PropTypes.func,
+  placeholder: PropTypes.string,
   step: PropTypes.number,
-  value: PropTypes.number,
-  onChange: PropTypes.func,
-  onPressIncrement: PropTypes.func,
-  onPressDecrement: PropTypes.func,
-  ...getDataProps(),
+  value: PropTypes.oneOfType([ PropTypes.string, PropTypes.number ]).isRequired,
```

Comment:
why can value be a string?

Reply:
im assuming this is because some places are passing string into the input? (ie. non-redux-form usages)